packages/sources/wootrade/README.md

```diff
@@ -0,0 +1,55 @@
+# Chainlink External Adapter for Wootrade
+
+Adapter using the public Wootrade market API for both HTTP(s) and WS.
+
+### Environment Variables
+
+| Required? |  Name   |                                                        Description                                                         | Options | Defaults to |
+| :-------: | :-----: | :------------------------------------------------------------------------------------------------------------------------: | :-----: | :---------: |
+|     ✅      | API_KEY | An key to use the wootrade WS API |         |             |
```

Comment:
Do you only need an API key to use the WS? Or HTTP as well?

Also, where do you sign up for an API key? I tried at https://x.woo.network/, but it's not letting me submit the signup form :confused:

Reply:
Only for ws, the documentation is here: https://kronosresearch.github.io/wootrade-documents/#introduction